src/wp-includes/kses.php

```diff
@@ -2438,6 +2438,13 @@ function safecss_filter_attr( $css, $deprecated = '' ) {
 			'overflow',
 			'vertical-align',
 
+			'position',
+			'top',
+			'right',
+			'bottom',
+			'left',
+			'z-index',
```

Comment:
Add document for this changes.

```
* @since 6.2.0 Added support for `position`, `top`, `right`, `bottom`, `left` and `z-index` position CSS properties.
```

Reply:
Good catch, thanks @mukeshpanchal27! I've added that line in https://github.com/WordPress/wordpress-develop/pull/3865/commits/08ccde650e44c66f327ef91103b3887b5c407705